docs/using/event-types.md

```diff
@@ -47,81 +47,90 @@ have a more specific definition :)
-backward compatible, in the sense that trying to read an older event
-using a newer schema is not safe.
+1. Consumers using older schemas can safely read events validated by
+   newer schemas.
+2. Clients using newer schemas can safely read events validated by
+   older schemas.
 
-For example, if the consumer has been developed with schema version
-`1.5.0` in mind, reading events from `1.5.0` on is safe, for example,
-`1.6.0`. But reading event generated by older schemas, like `1.4.0`,
-is not safe.
+It guarantees high data quality which is crucial for a variety of
+applications. At Zalando, it's required to use this compatibility mode
+to have data processed by business intelligence and long term event
+storage by the data lake.
 
 Supported changes:
 
-1. changes to meta attributes: titles, descriptions and examples.
-2. addition of new attributes, both optional and required one.
-3. marking attributes as required.
+1. changes to meta attributes: titles and descriptions.
+2. addition of new optional attributes.
 
-Consumers reading events from forward compatible event types SHOULD
-ignore event attributes not declared in the event type schema.
+The following json-schema attributes are not unsupported:
 
-This mode is default because it solves the most common requirement:
-guarantee the stability of deployed consumers applications when the
-schema changes. It's crucial for the robustness of event driven
-applications.
+- additionalProperties
+- additionalItems
+- not
+- patternProperties
 
-#### Compatible
+Removing the support for these attributes is necessary to avoid the
+introduction of incompatible changes.
 
-This is the most strict mode. It's both forward compatible and
-backward compatible. In practical terms:
+Under this compatibility mode it's necessary to fully specify events
+properties in order for validation to succeed. I.e. events with
+properties not declared in the schema will be rejected. For this
+reason, producers should first update their schemas with new
+attributes and only after that start producing events with such
+attributes.
 
-1. Consumers using older schemas can safely read events validated by
-   newer schemas.
-2. Clients using newer schema can safely read events validated by
-   older schemas.
+#### Forward compatible
 
-It's useful for long term data storage and analysis. It guarantees the
-highest quality data for business intelligence.
+The `forward` mode has been designed to allow event type owners to
+expand their schemas without breaking existing consumers.
 
-Only a subset of json schema is allowed, to make sure that no
-accidental incompatibilities or ambiguities are introduced. This mode
-is required for data to be stored and processed by the data lake.
+It's called forward because consumers using older schemas can safely
+read events generated by newer schemas. On the other hand, it's not
+backward compatible, in the sense that reading older events using
+newer schemas is not safe.
```

Comment:
editorial proposal:
...sense that reading older events using newer schemas is not safe. This is basically due to the fact that we allow introduction of additional mandatory attributes: 

I would delete the next paragraph -- example is IMO not helpful since text is already very clear.